src/documentation/markdown/2018/filing-faq.md

```diff
@@ -61,62 +61,58 @@ Additionally, if you are using Internet Explorer, we have found that filers have
  - ffiec.cfpb.gov
  - ffiec.beta.cfpb.gov
 
+## Submission
 
-### Submission
-
-#### When is the filing period?
+### When is the filing period?
 The **Annual filing** period is between <a target="_blank" rel="noopener noreferrer" href="https://ffiec.cfpb.gov/documentation/2018/annual-filing-dates/">January 1 and March 1</a> of each year.  The <a target="_blank" rel="noopener noreferrer" href="https://ffiec.cfpb.gov/filing">HMDA Filing Platform</a> for each filing period opens January 1st.  The submission deadline is March 1st.  
 
-#### How do I know my file was submitted?
+### How do I know my file was submitted?
 Upon signing and submitting your file, an email confirming your submission will be sent to the email address on file. Additionally, when you log into your account, you should see a message stating:
 "Your submission has been accepted. This completes your HMDA filing process for this year. If you need to upload a new HMDA file, the previously completed filing will not be overridden until all edits have been cleared and verified, and the new file has been submitted." 
 Below that message is a button that says 'View Completed Filing'. Select that and you will see the date and timestamp your file was accepted along with a receipt number for the submission. 
 
-#### When can I submit my file?
+### When can I submit my file?
 The <a target="_blank" rel="noopener noreferrer" href="https://ffiec.beta.cfpb.gov/">HMDA Beta Filing Platform</a> is open throughout the year to test your HMDA data.
 
-_Annual HMDA data submissions_ for the 2018 filing year are accepted January 1 - March 1, 2019.  
+_Annual HMDA data submissions_ for the filing year are accepted between January 1 – March to be considered timely. 
 <br/>
 
 ![2018 annual filing dates](https://raw.githubusercontent.com/cfpb/hmda-frontend/master/src/documentation/markdown/images/annual_filing.png)  
 
-#### Should I file quarterly?
+### Should I file quarterly?
 Beginning January 1, 2020, the 2015 HMDA Final Rule requires quarterly reporting for financial institutions that reported a combined total of at least 60,000 applications and covered loans, excluding purchased covered loans, for the preceding calendar year. Thus, in addition to their annual data submission, these larger-volume reporters will submit HMDA data for each of the first three quarters of the year on a quarterly basis. Further information can be found in <a target="_blank" rel="noopener noreferrer" href="https://www.consumerfinance.gov/rules-policy/regulations/1003/5/#a-1-ii"><a target="_blank" rel="noopener noreferrer" href="https://www.consumerfinance.gov/rules-policy/regulations/1003/5/#a-1-ii">Regulation C, 12 CFR 1003.5(a)(1)(ii)</a></a>. 
```

Comment:
Please double check if this occurs in other years as well.

```suggestion
Beginning January 1, 2020, the 2015 HMDA Final Rule requires quarterly reporting for financial institutions that reported a combined total of at least 60,000 applications and covered loans, excluding purchased covered loans, for the preceding calendar year. Thus, in addition to their annual data submission, these larger-volume reporters will submit HMDA data for each of the first three quarters of the year on a quarterly basis. Further information can be found in <a target="_blank" rel="noopener noreferrer" href="https://www.consumerfinance.gov/rules-policy/regulations/1003/5/#a-1-ii">Regulation C, 12 CFR 1003.5(a)(1)(ii)</a>. 
```

Reply:
This does occur in other years.

Reply:
Has been updated for all markdown 2018-2023.